src/common/types.ts

```diff
@@ -9,6 +9,12 @@ export type ColorPreset =
 
 export type CustomColor = string
 
+export interface CustomColorObject {
```

Comment:
Can we avoid having to use another type? how about this?
```javascript
export type CustomColor = { color: string, secondaryColor: string } | string
```

Reply:
thanks, it's better and clean